### ISSUE TITLE
Fatal error: Uncaught Error: Class "LP_Model_User_Can_View_Course_Item" not found

### ISSUE THREAD

**Issue by buonzz** [LearnPress/learnpress#516]
Hi,

I've use `[learn_press_course_curriculum id="9215"]`
in an elementor page. but when I edited the page, it shows the following error

```
Fatal error: Uncaught Error: Class "LP_Model_User_Can_View_Course_Item" not found in /var/www/html/wp-content/plugins/learnpress/inc/user/class-lp-user.php:20 Stack trace: #0 /var/www/html/wp-content/themes/eduma/learnpress-v4/single-course/tabs/curriculum.php(21): LP_User->can_view_content_course() #1 /var/www/html/wp-content/plugins/learnpress/inc/lp-template-functions.php(850): include('...') #2 /var/www/html/wp-content/plugins/learnpress/inc/shortcodes/class-lp-shortcode-course-curriculum.php(47): learn_press_get_template() #3 /var/www/html/wp-content/plugins/learnpress/inc/abstracts/abstract-shortcode.php(81): LP_Shortcode_Course_Curriculum->output() #4 /var/www/html/wp-includes/shortcodes.php(356): LP_Abstract_Shortcode->__toString() #5 [internal function]: do_shortcode_tag() #6 /var/www/html/wp-includes/shortcodes.php(228): preg_replace_callback() #7 /var/www/html/wp-content/plugins/wp-events-manager/inc/wpems-core-functions.php(152): do_shortcode() #8 /var/www/html/wp-includes/class-wp-hook.php(307): wpems_the_content() #9 /var/www/html/wp-includes/plugin.php(191): WP_Hook->apply_filters() #10 /var/www/html/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php(1811): apply_filters() #11 /var/www/html/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php(560): WP_REST_Posts_Controller->prepare_item_for_response() #12 /var/www/html/wp-includes/rest-api/class-wp-rest-server.php(1143): WP_REST_Posts_Controller->get_item() #13 /var/www/html/wp-includes/rest-api/class-wp-rest-server.php(990): WP_REST_Server->respond_to_request() #14 /var/www/html/wp-includes/rest-api.php(519): WP_REST_Server->dispatch() #15 /var/www/html/wp-includes/rest-api.php(2868): rest_do_request() #16 [internal function]: rest_preload_api_request() #17 /var/www/html/wp-includes/block-editor.php(601): array_reduce() #18 /var/www/html/wp-admin/edit-form-blocks.php(75): block_editor_rest_api_preload() #19 /var/www/html/wp-admin/post.php(187): require('...') #20 {main} thrown in /var/www/html/wp-content/plugins/learnpress/inc/user/class-lp-user.php on line 20
```

anyone knows what could be wrong?

**Comment by tungnxt89** [comment]
Hi buonzz,
We see that, we'll fix on the next version.

Thanks.

**Comment by tungnxt89** [comment]
Hi buonzz,
We fixed that on LP v4.1.6.9. Please check it.

Thanks.
